packages/web-components/src/wallet-connection/watchWallet.ts

```diff
@@ -33,6 +33,8 @@ type VBankAssets = [
 
 /** @typedef {import('@agoric/ertp/src/types.js').Amount<'nat'>['value']} NatValue */
 
+/** @typedef {import('@agoric/ertp/src/types.js').Amount<'nat'>['value']} NatValue */
```

Comment:
this is a `.ts` file so the jsdoc tyepdefs do nothing. 

this and the one above should be removed. Line 19 actually imports the type

Reply:
Oh yea this seems to be a merge artifact, removed.